fabric-dimensions-v1/src/main/java/net/fabricmc/fabric/api/dimension/v1/FabricDimensions.java

```diff
@@ -30,6 +31,7 @@
  *
  * @deprecated Experimental feature, may be removed or changed without further notice due to potential changes to Dimensions in subsequent versions.
  */
+@ApiStatus.Experimental
 @Deprecated
```

Comment:
I suggest we make this stable.

Reply:
Removing dimensions still destroys worlds though. Not sure if that can be considered stable, or not? Idk

Reply:
Thats a vanilla bug that has nothing to do with the API that is provided here (teleporting).

Reply:
Done.